Examples/MagicWeatherSwiftUI/Shared/Sources/Views/WeatherView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  WeatherView.swift
+//  Magic Weather SwiftUI
+//
+//  Created by Cody Kerns on 1/19/21.
+//
+
+import SwiftUI
+import Purchases
+
+/*
+ The app's weather tab that displays our pretend weather data.
+ */
+
+struct WeatherView: View {
+    @Binding var paywallPresented: Bool
+    @ObservedObject var model = WeatherViewModel.shared
+    
+    var body: some View {
+        VStack {
+            /// - Sample weather details
+            
+            Text("\(model.currentData.emoji)")
+                .padding(.top, 32.0)
+                .font(.system(size: 76))
+            Text("\(model.currentData.temperature)")
```

Comment:
could we add the unit? either to the model or just here when presenting?

Reply:
totally missed this- just pushed the fix 👍